backend/src/services/review.ts

```diff
@@ -141,6 +149,20 @@ export async function findReviewsForAccessRequests(accessRequestIds: string[]) {
   return reviews.filter((review) => requiredRoles.accessRequest.includes(review.role))
 }
 
+export async function findReviewForAccessRequest(accessRequestId: string) {
+  const review = await Review.findOne({ accessRequestId: accessRequestId })
+
+  if (!review) {
+    throw NotFound('The requested access request review was not found.', { accessRequestId })
+  }
+
+  if (!requiredRoles.accessRequest.includes(review.role)) {
```

Comment:
I'm a bit confused by the need to do this check if this function just needs to find existing reviews that relate to a specific access request?

Reply:
I took this from the existing functionality in the above `findReviewsForAccessRequests` function https://github.com/gchq/Bailo/blob/8f15cdfa16d6815642093db6145bd6748c8e2ed8/backend/src/services/review.ts#L149